pallets/collator-allowlist/src/benchmarking.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg(feature = "runtime-benchmarks")]
+use super::*;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+
+benchmarks! {
+	// Add a collator to the allowlist
+	add {
+		let collator_id = get_account::<T>();
```

Comment:
you would need to whitelist the account so that benchmarking wouldn't consider the key underneath for change

Reply:
I am not sure what you mean, sorry. Are you suggesting adding a new use case where the collator had already been added to the allowlist and thus we verify that we get an error in such case?

Reply:
Wdytm @vedhavyas? The whitelisting. Not sure I get it...

Reply:
Something like this - https://github.com/centrifuge/centrifuge-chain/blob/tocc/pallets/loan/src/benchmarking.rs#L88
Account needs to be whitelisted so that any changes to this account's balance etc are not tracked by the benchmark pallets unless we want that to be tracked. 
https://github.com/centrifuge/centrifuge-chain/blob/tocc/runtime/development/src/lib.rs#L962

Reply:
Thanks for the links, Ved. The advantage being that the benchmark doesn't take that collateral work into account when calculating the weights?

Reply:
yes

Reply:
Looking through this, it seems very useful to know about this but it doesn't seem quite as useful in this specific pallet's benchmarking. We don't play around with balances here and adding that extra code would add significant noise for a marginal - if any - difference or so it seems to me. what do you think, Ved?